packages/build-utils/src/fs/read-installed-version.ts

```diff
@@ -0,0 +1,17 @@
+import { readJSON } from 'fs-extra';
+import { cwd } from 'node:process';
+
+export async function readInstalledVersion(
+  pkgName: string
+): Promise<string | undefined> {
+  try {
+    const descriptorPath = require.resolve(`${pkgName}/package.json`, {
+      paths: [cwd()],
+    });
+    const descriptor = await readJSON(descriptorPath);
+    return descriptor?.version;
+  } catch {
+    // ignore errors: the package is simply not installed.
```

Comment:
Thanks for the tips.
Since `Client` interface is defined in cli package, and to avoid a circular dependency, I'm moving this utility function back to cli instead of build-utils.